test/cloudformation/iam_cloudformation.yaml

```diff
@@ -249,6 +249,24 @@ Resources:
           - Effect: Allow
             Action: timestream:DescribeEndpoints
             Resource: "*"
+  # This second is need to add more permission
+  # GithubActionsPolicy has hit the maximum character size limit for managed policies of 6,144
+  # https://repost.aws/knowledge-center/iam-increase-policy-size#
```

Comment:
Are we absolutely sure that we can't reduce the other policy? What if the other policy is smaller in the future, how do we reason about which policy should go where? If we absolutely need it, we should think of two reasonable policies groups, especially if we need to add more permissions to the other policy. If the current groups are the other one and this one, that's fine, but I'd like us to be more deliberate about this rather than just appending it to the list.

Reply:
I will look into a way to split up the first policy into multiple, to prevent us from hit the limit